pkg/cloudprovider/provider/gce/service.go

```diff
@@ -40,15 +44,23 @@ func connectComputeService(cfg *config) (*service, error) {
 
 // networkInterfaces returns the configured network interfaces for an instance creation.
 func (svc *service) networkInterfaces(cfg *config) ([]*compute.NetworkInterface, error) {
+	network := cfg.network
+
+	if cfg.network == "" && cfg.subnetwork == "" {
```

Comment:
I see I have to take a deeper look at GCE networking. How would it look like if network is empty and subnetwork has value? 🤔

Reply:
it will pass subnetwork only to the instance, which is OK

Reply:
Thx.